TAInformation/Models/account_type.py

```diff
@@ -19,3 +19,16 @@ def __str__(self):
             return "Instructor"
         elif self.value == self.ADMIN.value:
             return "Admin"
+
+    # pre conditions: 0-3 enum value
+    # post condition: returns an integer associated with the enum value
+    # side effects: raises an error if enum value is outside 0-3
+    def __int__(self):
+        if self.value == self.ADMIN.value:
+            return 3
+        elif self.value == self.TA.value:
+            return 1
+        elif self.value == self.INSTRUCTOR.value:
+            return 2
+        else:
+            return "Default"
```

Comment:
I think we want default value to be 0 bc if we used the string "default" it would throw errors bc we made it an Integer Field

Reply:
nice catch, that completely slipped by me. fixed